### DISCUSSION TITLE
Terminal not working in HTTPS

### DISCUSSION THREAD

**Discussion by sergedc** [0xJacky/nginx-ui#415] [Q&A] [answered]
**Describe the bug**
The terminal is working when I access the web-ui on port 9000 using http
The terminal is NOT working when I access the web-ui on port 9001 using https

**To Reproduce**
1. Setup https access for the web-ui:
server {
    listen 9001 ssl;
    server_name subdom.myweb.com;
    ssl_certificate /root/cert/fullchain.cer;
    ssl_certificate_key /root/cert/subdom.myweb.com.key;
    location / {
        proxy_pass http://localhost:9000;
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
    }
}

2. Access the web-ui on https://subdom.myweb.com:9001
3. Got to terminal: instead of "my-server login:" , I see nothing: the same black terminal box but no text inside

**Screenshots**
![image](https://github.com/0xJacky/nginx-ui/assets/114958550/11fd6c81-19a3-4944-a37e-7c3df134ae0d)

**Info (please complete the following information):**
 - Server OS: Ubuntu 24.04
 - Server Arch: x86
 - Nginx UI Version: 2.0.0-beta.24 (135)
 - Your Browser: Chrome

**Comment by 0xJacky** [comment, marked as answer]
Hi @sergedc, this is not a bug of Nginx UI. You have to configurate the nginx to proxy websocket connections to nginx ui.
Please check this configuation example for more detail https://nginxui.com/guide/nginx-proxy-example.html

**Comment by sergedc** [comment]
Thanks, very stupid of me to have missed this.
I was wondering is there a way to access the webui on hppts directly? Ie to make port 9000 https rather than forward to http 9000

**Reply by 0xJacky** [reply]
Sorry, not supported yet.
